content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,108 @@ aliases:
 
+<!-- TODO summarize effects on other resources -->
+
+## Before you begin: important considerations
+
+Before upgrading to InfluxDB 2.0, please consider the following issues;
+some or all might apply to your specific installation and use case.
+The sections below contain our recommendations for addressing possible gaps in the upgrade process.
+You may also find while reading that you'd like to hold off on upgrading for now.
+
+### Continuous queries
+
+When upgrading from InfluxDB 1.x and using continuous queries (CQs) to downsample data,
+we recommend extracting CQs from your InfluxDB 1.x instance prior to upgrade with `show continuous queries`.
+Run the following:
+
+```sh
+$ influx
+Connected to http://localhost:8086 version 1.8.3
+InfluxDB shell version: 1.8.3
+> show continuous queries
+```
+
+Save this output so it can be used later.
+
+Continuous Queries are replaced by Tasks in InfluxDB 2.0
+To convert your CQs to Flux, see the [Tasks documentation](/influxdb/v2.0/process-data/get-started/).
+
+### Supported protocols
+
+If you're using one of the [supported protocols](/influxdb/v1.8/supported_protocols/) in InfluxDB 1.x (CollectD, Graphite, OpenTSDB, Prometheus, UDP),
+you may want to research using Telegraf instead.
+InfluxDB 2.0 doesn't support these protocols directly.
+Use Telegraf to translate from the source protocol to InfluxDB 2.0.
+
+### Kapacitor
+
+To continue using Kapacitor, update the Kapacitor configuration to use the appropriate security credentials per [1.x compatibility API docs 1]()
+(using the appropriate user and token to allow access).
+
+### Dashboards
+
+There is no [subscriptions API](/influxdb/v1.8/administration/subscription-management/) in InfluxDB 2.0.
+To continue to use existing dashboarding tools such as Chronograf and Grafana with InfluxDB 2.0,
+reconfigure your data sources to point to your new InfluxDB 2.0 OSS instance with the appropriate security credentials.
+
+### User migrations
+
+`influxd upgrade` attempts to migrate existing 1.x users and their permissions.
+However, *administrative users are not copied*.
+
+If using an admin user for vizualition or Chronograf administrative functions, you may want to create a new read only user first.
+Admin capabilities are to the primary user created in the 2.0 setup process via the `influxd upgrade` command.
+
+### Note on platforms
+
+InfluxDB 2.0 is currently available for MacOS and Linux.
+Docker images, ARM builds, and Windows builds are not currently available.
+
+<!-- TODO? 1.x Compatibility APIs - dashboards, client libraries and other -->
+
+## Perform the upgrade
+
+If you've considered the issues above in [Before you begin](#before-you-begin-important-considerations) and are ready to proceed, follow these steps to upgrade your InfluxDB 1.x to InfluxDB 2.0.
+
 {{% warn %}}
-Back up all data before upgrading with `influx upgrade`.
-{{% /warn %}}
+Back up all 1.x data before upgrading with `influx upgrade`.
```

Comment:
let's not use back up.... make a copy?  Essentially, we recommend ensuring that you have a copy of all of your data prior to starting the process.

Reply:
They should make this copy once they stop the `influxd` process